src/utils/presets.ts

```diff
@@ -213,13 +213,30 @@ export const uiliciousPreset = {
BS6YvjB6mGFIM5EQBZ0gqHBo7Hn3qG+Y7i1OMo+vq+WrUqgKMIIHCetmTp4Bw3Q4pDexaAXtBmVRJue8AIAgjM0AF1bcmiGlIs2rMAtIttVqUZScLHhr216g4BDCCAwJRGNSSOH0x+WBEUKt2epUGEqgiAFNWOkkVRtV/k6wyawxICCExqXVDXAXXW9ZZAKyJfvOTzjTcAYaLaUTrW68IsAgjMYl1v+ZgVAcKi9zrS2x1UO0pE1QPGEUBgHut63aBB5IpWZWTZwFIB4If0YOAvd71AtaN8VD3gBQIIvEA1xC20aAH2pXc79D3VDgdQ9YBHCCDwCtUQt6QtWmzRAmzQsKF3O7TiQehwBlUPeIcAAu+wKctNGkb+JKmIaBg579yzBYAbNHRoaxVzHQ6i6gFPEUDgLe6GuIswApQrDR16ofz1/QcEzqHqAa8RQOA1rqi7jzACFIPQYQRVDwSAAIIgUA2xgZkRIFvMdJhST6oeX0+qHlUBPEcAQTC0GjImclNfHH9b4Lw0jOgmLQ0jrPYF5qYhI12Xq8GD0GHDRBT9WKv19Vp1RIAAEEAQHFb22qRh5AtJGPnj5D2tWsDH9Djgs0nY4E6HQVFUTb4XbaHqgdAQQBCss4crjWoIQcQeqiMImVY1Xnl1X6PCoW/Mc5jEkDmCRgBB0BhS90NaHflUUhlhdgQ+SgfI0/cwLJIdYxJtoN0KISOAANIYUh+O4vhxqiH2aTVEA8n/nrzRrgWrtMLxD0nQ+L9bgYNZDi/UkqrHBtqtAAIIcALasvxDIIEFBA6v0W4FTEEAAaagLctvGkjOT0IILVsoSzrD8dvkjcDhN7ZbAdMjgAAzaASROH48+SFPqJ7TCsn5rTBy5uqVVEmQKd1SpbMbLyeB48XWe3iO7VbArAggwBw4YhietEryR61gooFEgwkwFw0bGjA0cOiPqW4Ep56Ejy17a9UdAmBGBBCgTWcPV26PmkcMBwXBmRpKqJRgcmWDsBE8nfP48XGRe2m3AuZGAAE6wHwIJktDyZpWGPlUq1rCXRK/pFUNvbehsxtp4CBsQDHnAXSOAAJ0gSCC2UwNJlotoWLiPoIGOsKcB9A1AgjQg7OHK5W+OH6Q+RC06/xWhUTfr24FE33TH1M5yZcGif2tkPF+axPVaPI+DR1AWwgeQM8IIEAGGFRHFjSArGkFEQ0oA8l7DSfpf0dImZmGCw0TWrHQt9db79OAoX+fSgZ6xIA5kBECCJAhggiKMDmUDLRCi75f3vqxSv87q4ElDRSjreCgYeL9SSFDpWGDcIGcNQbMk4rH7QIgEwQQIAe6Masvjq8liMAFy1pBZHIgSf9arZmyYni2lcNrpvl7aViYyeT2ptEpYWFymEj/XYQJOILNVkBOCCBATnRQPfnGtY7VvQBgCsEDyBkBBMgZG7MAwASCB1AQAghQEIIIADiJ4AEUjAACFIwgAgBOIHgAJSGAACUhiABAKQgeQMkIIEDJ0mF1tmYBQK4IHoAjCCCAI1oVkQp3RAAgUwQPwDEEEMBBHDQEgJ7Vk+Cxk+ABuIcAAjiMIAIAHYqi6oTIj39Xqz4hAJxEAAEMOHu4Uokk3iyxVAQAcLIkeCQv1mzZV6tWBYDTCCCAIWzOAoATjCTBY2cSPJ4geAB2EEAAg9icBSBwDJYDhhFAAON0TiR59+2kKjIsAOCzKKom/7lzLKl4EDwAuwgggCeacyKyjvYsAL6JougJHSynzQrwAwEE8EyrPevyvjj+Nu1ZAAyjzQrwFAEE8FizPSupiLA9C4AVbLMCvEcAAQKQbs+K4vhiqiIAHES1AwgIAQQIDFURAM6g2gEEiQACBEqrIsk3/stjZkUAFItqBxA4AgiAyRu0Lkv+clAAIFscDATwEQIIgI8MDVcG5ydVEVq0AGQiiqp9Ik8eFdlBtQNAigACYFqtwfWKcOQQQCeacx3P02IFYCYEEABzOme4ogFkXRTHlzEvAmAajbmO5H2VFisAcyGAAOhIOi/CSl8geMx1AOgKAQRA1wgjQHAIHQB6RgABkAnCCOAtQgeATBFAAGSuFUYuZ2YEsCmKovqEyE5hpgNADgggAHKVHjyckPgyVvsCDmutzE0eDJ54pVatCwDkhAACoDDpat8oCSNxM4xw9BAoz0jyFPBE8ijw/FgSOliZC6AoBBAApUlbtZIfXsytEaAAUVRL/vN55jkAlIkAAsAJVEeAXDSqHH0S/YZr5ABcQQAB4CStjvSLDDM7AnREN1bV9BK5MEAOwFEEEAAmfNyuFX+aQAJMEkXV5D9/o21Vx0VqVDkAuI4AAsCcoeHK4LykOkIgQZAIHACMI4AA8IJWSPpEKrHEFyeBRAfamSGBDxotVboedzwJGwQOAD4ggADw0jnDFQ0h+lZJ3j7Nli1YoAcAkxBd1aHxCZHq3lq1JgDgGQIIgCCkbVt9jUDSaNsa5ko7yqRhI/lcrEUSPU91A0BICCAAgqWrf8eSUNLfqJQQSpCfyWEjCRr1JHBUCRsAQkUAAYBJ0kqJhhKdJ4klGqJ9Cx3QmY26hg1to6KyAQAnI4AAQBt0pmQiySdptYRggnReI6lqvJZUzurJ50b1lVq1LgCAWRFAAKAHGkySh8/B5tFEbd9qtnIJW7i80AwZSVWjVdHQ9qn5SUWDoAEA3SOAAEAOtJUreVAd0lAyr/W+EU4kGqRy4pSRJGQ0AkZS1XqtLwkYGjL0/VjyRusUAGSPAAIAJUhnTbR6kgaUSOKz9K8jiYbiOB4S9GxyBSP5q/c1WEw0/lpqSdVqhEoGABSPAAIAjkqrKM1QIkPJw/Ng3HiLz5LWtq5WWNF2r1BavhoVi+T3oJ78noxouNAZDP3xRPOv60mgqx9Jfkz1AgDcRAABAE9oYFmUBJHjrXCiYaWv9Ra3AkorvHz0P5n8v9cwM/Xf2W0lprl2dsq/q1mJmBwK6pP+v19rvm8GidY/3wgZWqkgUACAP/5/FUgSq4zSx4YAAAAASUVORK5CYII=',
```

Comment:
hello @zainfathoni do you wanna point the image to an external image link instead in case if you ever wanna update this? I find it a bit funny to just put raw `data:image` here in this file hehe

Reply:
Haha 😅
Yes, it's weird indeed. 😁

Sure, I'll do that later. Thanks!

Reply:
Updated https://github.com/lyqht/styled-qr-code-generator/pull/12/commits/56f3d550932ecad4ed10dec00349b7213d36bf5b.
Thanks!